### ISSUE TITLE
ControlAction participant

### ISSUE THREAD

**Issue by ChristiaanScheermeijer** [trompamusic/ce-api#103]
@alastair @musicog for the Campaign Manager we would like to store the username of the person who is contributing by doing some ControlActions. 

The ControlAction has the `agent` and `participant` properties which seems to be the right place to put this kind of information. However, both properties should be of type Person or Organization according to schema.org.

What do you think?

**Comment by alastair** [comment]
In the modelling discussion (https://docs.google.com/document/d/1B1O2tqaubbEHzXSWhvSDYqYu2lwdNKg4Vy04ukMfrUU/edit?pli=1) we discussed this a little bit for the Annotations use-case.
We're proposing that we use a user's vcard uri for the `creator` field. The vcard uri is the uri that identifiers a user on a solid pod.
Can you use something similar for ControlActions?

**Comment by ChristiaanScheermeijer** [comment]
The ControlAction is only implementing schema.org properties, can we safely implement the Dublin core metadata interface as well (where the creator property is defined)?

**Comment by alastair** [comment]
We included [dc items in the Annotation schema too](https://github.com/trompamusic/ce-api/blob/3e5abef134af8e784fc7c31fdb17dea17cffb060/src/schema/type/Annotation.graphql#L88-L92). @musicog said that if there's no equivalent in schema.org then it's OK to pull in specific items from dc if needed.
Note that this also ties into #96 a bit too, deciding which dc schema we want to use. I see that I use dcterms for creator and dcelements for created/modified, and there's also a bug in the way that I specified created, and the namespaces [are different in DefinedTerm](https://github.com/trompamusic/ce-api/blob/9bf4d13203399440fe14559300f70802a8f9c6b4/src/schema/type/DefinedTerm.graphql)... I'll go and clean these up, but we have to make a final decision on other items too.
